apsuite/orbcorr/orbit_correction.py

```diff
@@ -202,20 +213,27 @@ def _process_kicks(self, dkicks):
         que = [(-par.maxkickch - kickch) / dkickch, ]
         que.append((par.maxkickch - kickch) / dkickch)
         que = _np.max(que, axis=0)
-        dkickch *= min(_np.min(que), 1.0)
+        coef_ch = min(_np.min(que), 1.0)
 
         que = [(-par.maxkickcv - kickcv) / dkickcv, ]
         que.append((par.maxkickcv - kickcv) / dkickcv)
         que = _np.max(que, axis=0)
-        dkickcv *= min(_np.min(que), 1.0)
+        coef_cv = min(_np.min(que), 1.0)
 
         if self.params.enblrf and dkickrf != 0:
             que = [(-par.maxkickrf - kickrf) / dkickrf, ]
             que.append((par.maxkickrf - kickrf) / dkickrf)
             que = _np.max(que, axis=0)
-            dkickrf *= min(_np.min(que), 1.0)
+            coef_rf = min(_np.min(que), 1.0)
+
+        min_coef = min(coef_ch, coef_cv, coef_rf)
```

Comment:
I think this option of a global `min_coef` should be an option of the class, not a mandatory behavior. I think this is important because we can recover the behavior implemented in the SIRIUS SOFB algorithm.